src/Table/TableHead.js

```diff
@@ -84,11 +85,18 @@ class TableHead extends Component {
       }
     )
 
+    let thWidth
+
+    if (width >= 0) {
+      thWidth = width
```

Comment:
Creio que aqui seja melhor tratar o estilo todo, para que possa ser reutilizado da mesma maneira na linha 124.
```suggestion
      thStyle.width = `${width}px`
```

Reply:
Obrigado. Passou despercebido por mim. Resolvido.